aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -122,18 +123,18 @@ def __init__(
         self.max_age = max_age
         self.allow_credentials = allow_credentials
 
-    def to_dict(self) -> Dict[str, str]:
+    def to_dict(self) -> Dict[str, List[str]]:
         """Builds the configured Access-Control http headers"""
-        headers = {
-            "Access-Control-Allow-Origin": self.allow_origin,
-            "Access-Control-Allow-Headers": ",".join(sorted(self.allow_headers)),
-        }
+        headers: Dict[str, List[str]] = defaultdict(list)
+        headers["Access-Control-Allow-Origin"].append(self.allow_origin)
+        headers["Access-Control-Allow-Headers"].append(",".join(sorted(self.allow_headers)))
```

Comment:
np: do we need to sort it? previous iteration I know. It could save some cycles if unnecessary.

Reply:
as discussed, we are keeping the sort to facilitate not having to change all the tests